app/views/content/landing/inspire-the-next-generation/_content.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">How much could you earn as a teacher?</h2>
+
+    <div class="inset">
+      <p>
+        You will earn at least £28k (or more in London) if you're just starting as a teacher. The average salary for a classroom teacher in England is £39,500 a year.
+      </p>
+
+      <p>
+        There are other great benefits too, including a substantial pension, and generous holidays.
+      </p>
+
+      <p>
+        <%= link_to("Learn more about the benefits of a career in teaching", page_path("salaries-and-benefits")) %>.
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Can I get help funding my training?</h2>
+
+    <div class="inset">
+      <p>
+        Postgraduate teacher training course fees are around £9,250 per year.
+        You can apply for tuition fee and maintenance loans, even if you already have a student loan.
+      </p>
+
+      <p>
+        If you're interested in teaching certain subjects, you might be able to get a scholarship or bursary of up to £29k tax-free.
+        This is to support you while you are training and does not need to be paid back.
+      </p>
+
+      <p>
+        <%= link_to("Find out about funding your teacher training", page_path("funding-and-support")) %>.
+    </div>
+  </section>
+
+    <section class="col col-720">
+    <h2 class="heading--box-blue">How do I apply for teacher training?</h2>
+
+    <div class="inset">
+      <p>
+        Full-time postgraduate teacher training courses usually take 9 months,
+        and undergraduate courses can take up to 4 years.
+      </p>
+      <p>
+      Most courses start in September. You can start applying now for a place after you graduate.
+      </p>
+
+      <p>
+        <%= link_to("Find out how to train to teach", page_path("train-to-be-a-teacher")) %>.
```

Comment:
Feels a bit strange to be talking about applying in the header, but then we link them somewhere that isn't really about applying. If we don't want to link straight to Apply and want to keep them in GIT, could it be better to send them to our how to apply page instead?